EIPS/eip-1.md

```diff
@@ -419,41 +546,41 @@ If you are interested in assuming ownership of an EIP, send a message asking to
 
 The current EIP editors are
 
-- Alex Beregszaszi (@axic)
-- Gavin John (@Pandapip1)
-- Greg Colvin (@gcolvin)
-- Matt Garnett (@lightclient)
-- Sam Wilson (@SamWilsn)
-- Zainan Victor Zhou (@xinbenlv)
-- Gajinder Singh (@g11tech)
+* Alex Beregszaszi (@axic)
+* Gavin John (@Pandapip1)
+* Greg Colvin (@gcolvin)
+* Matt Garnett (@lightclient)
+* Sam Wilson (@SamWilsn)
+* Zainan Victor Zhou (@xinbenlv)
+* Gajinder Singh (@g11tech)
 
 Emeritus EIP editors are
 
-- Casey Detrio (@cdetrio)
-- Hudson Jameson (@Souptacular)
-- Martin Becze (@wanderer)
-- Micah Zoltu (@MicahZoltu)
-- Nick Johnson (@arachnid)
-- Nick Savers (@nicksavers)
-- Vitalik Buterin (@vbuterin)
+* Casey Detrio (@cdetrio)
+* Hudson Jameson (@Souptacular)
+* Martin Becze (@wanderer)
+* Micah Zoltu (@MicahZoltu)
+* Nick Johnson (@arachnid)
+* Nick Savers (@nicksavers)
+* Vitalik Buterin (@vbuterin)
 
 If you would like to become an EIP editor, please check [EIP-5069](./eip-5069.md).
 
 ## EIP Editor Responsibilities
 
 For each new EIP that comes in, an editor does the following:
 
-- Read the EIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to get to final status.
-- The title should accurately describe the content.
-- Check the EIP for language (spelling, grammar, sentence structure, etc.), markup (GitHub flavored Markdown), code style
+* Read the EIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to get to final status.
+* The title should accurately describe the content.
+* Check the EIP for language (spelling, grammar, sentence structure, etc.), markup (GitHub flavored Markdown), code style
 
-If the EIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+If the EIP isn't ready, the editor will send it back to the author with instructions for revision and possibly a request for review by Technical Peers. 
```

Comment:
```suggestion
If the EIP isn't ready, the editor will send it back to the author with instructions for revision and possibly a request for review by Technical Peers.
```

Reply:
+1 to the suggestion. Would it make sense to get more specific and recommend a particular WG that they get review from as well?